File: programming/answers/C/@mykeels/03-word-in-reverse.c
#include <stdio.h>
#include <string.h>

int main() {
    while (1) {
        printf("Enter a word: ");
    
        char word[50];
        scanf("%s", word);

        for (int i = strlen(word) - 1; i >= 0; i--) {
            printf("%c", word[i]);
        }

        printf("\n");
    }
    
    return 0;
}

File: answers/Go/@codehakase/03-word-in-reverse.go
package main

import "fmt"

func main() {
	var word string
	for {
		fmt.Print("Enter a word: ")
		fmt.Scanf("%s\n", &word)

		for i := len(word) - 1; i >= 0; i-- {
			fmt.Printf("%c", word[i])
		}
		fmt.Println()
	}
}
